docs/client.md

```diff
@@ -0,0 +1,156 @@
+# Catalyst Client
+
+Catalyst allows you to retrieve info from BigCommerce's APIs through the Catalyst Client.<br /><br />
+
+```ts
+import { client } from '~/client';
+```
+
+The client simplifies what you need to do to handle requests and responses. For example, you don't need to construct request URLs, configure request headers, or parse JSON responses. The client uses the channel ID and authorization tokens configured in the `.env` file by default.
+
+## Methods
+
+- `fetch()`: allows you to interact with BigCommerce's [GraphQL Storefront API](https://developer.bigcommerce.com/docs/storefront/graphql). You can execute all queries and mutations available in [BigCommerce's Storefront graph](https://developer.bigcommerce.com/graphql-storefront/reference).
+
+- `fetchSitemapIndex(channelId?: string)`: fetches the sitemap index for the store, which provides a URL to the XML sitemap.
+
+- `getCanonicalUrl(channelId?: string)`: constructs and returns the canonical URL for the BigCommerce store. This URL is used for constructing various API endpoint URLs and is based on the store's hash and the provided or default channel ID.
+
+- `getGraphQLEndpoint(channelId?: string)`: returns the GraphQL endpoint URL for the store, incorporating the optional channel ID to specify which store channel's GraphQL API to access. This URL is used for sending GraphQL queries and mutations.
+
+- `requestLogger(document: string)`: provides a logging function to help debug and monitor performance. Logs the operation type, name, request duration, and query complexity.
+
+### Deprecated
+
+The following method is deprecated and requires the `BIGCOMMERCE_ACCESS_TOKEN` to be set in the `.env` file.
+
+- `fetchShippingZones()`: retrieves shipping zones for the BigCommerce store. It sends an HTTP GET request to the [BigCommerce Management API](https://developer.bigcommerce.com/docs/rest-management/shipping-v2/shipping-zones), but simplifies how you would make a direct request to fetch shipping zones.
+
+## Fetch
+
+The `fetch()` method lets you send GraphQL queries or mutations to the [GraphQL Storefront API](https://developer.bigcommerce.com/docs/storefront/graphql).
```

Comment:
Just rephrased it to make it less repetitive. If I remove the sentence, it looks like the `## Fetch` section could be referring to `fetchSitemapIndex` or `fetchShippingZones()` (rather than just the `fetch()` method)